tests/test_visitors/test_ast/test_general/test_comparison_order.py

```diff
@@ -0,0 +1,253 @@
+# -*- coding: utf-8 -*-
+
+import pytest
+
+from wemake_python_styleguide.violations.consistency import (
+    ComparisonOrderViolation,
+)
+from wemake_python_styleguide.visitors.ast.order import WrongOrderVisitor
+
+# templates to be checked
```

Comment:
Single line comments should start with uppercase letters.
Maybe that should be included as a rule? 🤔

Reply:
Done! https://github.com/wemake-services/wemake-python-styleguide/issues/225